testdata/testpkg2/test3.proto

```diff
@@ -26,8 +27,8 @@ message Test3 {
     }
 
     oneof OneofField {
-        string oneoffield1 = 9;
-        string oneoffield2 = 10;
+        string oneoffield1 = 10;
```

Comment:
In general, we don't want to modify the field if we can (https://protobuf.dev/programming-guides/dos-donts/), as it can break serialization. In this instance, these protos aren't really used anywhere so it's not really a big deal, i just figured I would call it out. When adding a new field, we just select the next number that is free and use that

Reply:
Thought as much since this is only for testing, but you're right. I'll stick to the best practices